test/api/heavy_version_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+// +build heavy_mock_integration
+
+package api
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/insolar/insolar/ledger/heavy/exporter"
+	"github.com/insolar/insolar/pulse"
+	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/block-explorer/instrumentation/converter"
+	"github.com/insolar/block-explorer/test/heavymock"
+	"github.com/insolar/block-explorer/test/integration"
+	"github.com/insolar/block-explorer/testutils"
+)
+
+func TestGBEVersion_Error(t *testing.T) {
```

Comment:
as discussed, the whole file might be deleted

Reply:
done